.gitignore

```diff
@@ -1,6 +1,11 @@
 /dist
 /_output
 
+# For testing
+staging/k8s.io/kubernetes
```

Comment:
idk if "staging" makes sense here. I understand "staging" in k8s.io/kubernetes to mean that the code there is "staged" there before getting auto-mirrored to elsewhere. for example k8s.io/kubernetes is the source of truth for k8s.io/api so they do some go module magic to depend on k8s.io/api while also keeping the source of truth under staging dir . https://github.com/kubernetes/kubernetes/blob/master/go.mod#L482 .

some other name to indicate it is tmp for testing  would be better imo.. like testartifacts or sth.